apis/capabilities/v1beta1/product_types.go

```diff
@@ -224,7 +224,8 @@ type AppKeyAppIDAuthenticationSpec struct {
 	// +optional
 	Security *SecuritySpec `json:"security,omitempty"`
 
-	// TODO GATEWAY RESPONSE
+	// +optional
+	GatewayResponse *GatewayResponseSpec `json:"gatewayResponse,omitempty"`
```

Comment:
Is this independent on how `ProductDeploymentSpec` is configured?

Reply:
I think this comment answers my question: https://github.com/3scale/3scale-operator/pull/528/files?file-filters%5B%5D=.go&file-filters%5B%5D=.md#diff-bb6a1b34101270566aae5073dcb3e50543c2058eea30c95cc18957f033dbb0b4R585

I understand then it is independent and both modes accept the same set of fields in System